malaria24/ona/models.py

```diff
@@ -657,8 +657,15 @@ class ReportedCase(models.Model):
     def get_data(self):
             '''JSON Formats need create_date_time & date_of_birth
             to be overridden'''
-            birth_date = datetime.strptime(self.date_of_birth,
-                                           "%y%m%d")
+            try:
+                dob = datetime.strptime(self.date_of_birth, '%Y-%m-%d')
```

Comment:
`date_of_birth` would be preferable to `dob` as a variable name because it's more explicit.

Reply:
thanks Nathan, will amend that now